### DISCUSSION TITLE
I have a question.

### DISCUSSION THREAD

**Discussion by C0sm0s-Chill** [Pryaxis/TShock#2578] [Q&A]
Hello , is it possible to sperare the id of the npc of the negative ? to be able to configure loot to each npc
 Thanks ! :)

have a good day.

**Comment by Arthri** [comment]
It depends on which you are talking about. Generally speaking, yes, the `netID` of an NPC can be negative. https://terraria.fandom.com/wiki/NPC_IDs. However, those negative IDs are simply translated to positive npc `type`s with certain properties a bit different. NPC `type`s can't be negative.

**Comment by C0sm0s-Chill** [comment]
Let's say I use the 
LootTableEditor plugin and when I put a loot on the blue slime which has the Id "1" All the other slimes have the same loot .

**Comment by Arthri** [comment]
I don't know how LTE handles it but I believe no. You have do it manually for each mob
